plugins/plugin-chart-table/src/types.ts

```diff
@@ -74,16 +74,18 @@ export interface TableChartProps extends ChartProps {
     pageSize?: number;
     currentPage?: number;
   };
+  cachedChanges: any;
+  setCachedChanges: Function;
```

Comment:
Could we have the full signature here? See e.g. `SetDataMaskHook` type

Reply:
no need it here, removing